src/index.ts

```diff
@@ -1,5 +1,6 @@
 /* ===== General Stuff ===== */
 export {default as Assets} from './assets';
+export type {emojis as EmojisAssetsType} from './assets/emojis';
```

Comment:
I'm assuming you'll use this in private, how?

Reply:
We have the casting on the assets object in the private repo. When i'll have this type I can add it to the case and it will be added to the type.